package.json

```diff
@@ -37,18 +37,19 @@
     "@embroider/addon-shim": "~1.9.0",
     "@embroider/macros": "~1.16.0",
     "@embroider/util": "^1.0.0",
-    "babel-plugin-ember-template-compilation": "2.2.1",
-    "ember-cli-babel": "$ember-cli-babel",
     "ember-modifier": "^3.2.7",
-    "moment": "^2.29.1"
+    "ember-template-imports": "^4.3.0",
```

Comment:
Can't we just update the editor package or does that cause other issues?

We can't override it like this since eslint-plugin-ember depends on v3. If we keep the override, we need to specifically override the one from the ember-rdfa-editor, similar to what you did for ember-concurrency.

Reply:
Yes I was about to do that next, just wanted to pin to have everything working first